processing/src/test/java/org/apache/druid/query/TestBufferPool.java

```diff
@@ -147,4 +147,10 @@ public Collection<RuntimeException> getOutstandingExceptionsCreated()
   {
     return takenFromMap.values();
   }
+
+  @Override
+  public int getUsedBufferCount()
+  {
+    return Integer.MAX_VALUE;
+  }
```

Comment:
We can probably have the correct implementation for this class, even if it is a test utility.